Content.Server/Botany/Systems/MutationSystem.cs

```diff
@@ -138,86 +140,84 @@ public SeedData Cross(SeedData a, SeedData b)
     private void MutateFloat(ref float val, float min, float max, int bits, int totalbits, float mult)
     {
         // Probability that a bit flip happens for this value.
-        float p = mult*bits/totalbits;
+        float p = mult * bits / totalbits;
         p = Math.Clamp(p, 0, 1);
         if (!Random(p))
         {
             return;
         }
 
         // Starting number of bits that are high, between 0 and bits.
-        int n = (int)Math.Round((val - min) / (max - min) * bits);
+        int valInt = (int) MathF.Round((val - min) / (max - min) * bits);
```

Comment:
valInt isn't any more descriptive than n, and the purpose of n is explained in the comment above. I think this is fine though, it just goes to show that what you judge as a nondescript variable name is legible to me and vice versa.